packages/devtools_app/lib/src/analytics/_analytics_web.dart

```diff
@@ -557,6 +558,10 @@ Future<void> computeUserApplicationCustomGTagData() async {
   if (serviceManager.connectedApp.isDartWebAppNow) {
     userAppType = appTypeWeb;
   }
+  if (serviceManager.connectedApp.isFlutterWebAppNow) {
+    userAppType = appTypeFlutterWeb;
+  }
```

Comment:
can we put all of these checks into an if else block to make sure nothing is getting mistakenly overwritten in the future? also, in the default case, we should return 'dart_cli_app'.